data/relup_bases.eterm

```diff
@@ -0,0 +1,31 @@
+%% -*- mode: erlang; -*-
+
+#{<<"4.4.0">> =>
+      #{from_versions => [],otps => [<<"24.1.5-3">>,<<"23.3.4.9-3">>]},
```

Comment:
It's pretty clear that we'll only release one OTP version going forward (with the relup trick).
```suggestion
      #{from_versions => [],otp => <<"24.1.5-3">>},
```